dev-libs/processor-trace/processor-trace-2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Intel(R) Processor Trace decoder library"
+HOMEPAGE="https://github.com/01org/processor-trace"
+SRC_URI="https://github.com/01org/processor-trace/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/2"
+KEYWORDS="-* ~amd64"
```

Comment:
Hmm, I'm sorry if I asked this already but is there really a reason for `-*`? Does it work only with direct CPU access, or can it work on e.g. some data saved into a file and copied to non-x86 host? ;-)

Reply:
> Does it work only with direct CPU access 

Yes, it uses perf_event interface for get data from processors.